### ISSUE TITLE
problem with face mesh /ml5js

### ISSUE THREAD

**Issue by gutesbild** [processing/p5.js#6952]
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [X] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### p5.js version

js/1.1.9/p5.js

### Web browser and version

Version 16.6 (18615.3.12.11.2)

### Operating system

Mac OS 13.6.3 

### Steps to reproduce this

(important note: I'm an artist, not a professional coder— please forgive me if my descriptions aren't precise).
I made a p5.js sketch that recognises points in a face and uses these points to make a drawing that reacts to movements.
The sketch worked for months, and just stopped working in the last two days.

My sketch is rather complex, but I found this example from somebody else, that also seems to have stopped working ( Error code "TypeError: Load failed"
https://editor.p5js.org/tlsaeger/sketches/bGBDeBsVv
--
my suspicion is that there is a problem loading the ml5js library?

Thank you for your help!

**Comment by wwwld1** [comment]
Hi @gutesbild , as I tested with the [link](https://editor.p5js.org/tlsaeger/sketches/bGBDeBsVv) from your issue, it works on my computer. I am using a Macbook(macOS 12.6). I tested on Chrome(Version 123.0.6312.123), Firefox(124.0.2) and Safari(16.0 (17614.1.25.9.10, 17614)) and they all works.
I also added a screenshot to show how it works.
<img width="1455" alt="image" src="https://github.com/processing/p5.js/assets/116049361/d2fa1a78-ed9c-4be7-83bc-a64279a9b52c">
I think maybe it is a problem with the version of your OS or browser, using some other browsers may solve it
